OpenStreetMapViewer/src/main/java/org/osmdroid/samplefragments/events/SampleLimitedScrollArea.java

```diff
@@ -43,36 +43,87 @@ public String getSampleTitle() {
 	// Constructors
 	// ===========================================================
 
+	private final Polyline mNorthPolyline = new Polyline();
+	private final Polyline mSouthPolyline = new Polyline();
+	private final Polyline mWestPolyline = new Polyline();
+	private final Polyline mEastPolyline = new Polyline();
 
 
 	@Override
 	protected void addOverlays() {
 		super.addOverlays();
 
-		final Polygon polygon = new Polygon();
```

Comment:
this whole block goes from a polygon to a set of lines. is there a particular reason?

Reply:
Definitely: now you can set limits for latitude only and for longitude only. And for both. Therefore the Polygon was not relevant anymore, and was replaced by 2 times 2 Polylines as min/max limits.